pkg/netceptor/netceptor.go

```diff
@@ -31,6 +31,9 @@ const ServiceAdTime = 60 * time.Second
 // SeenUpdateExpireTime is the age after which routing update IDs can be discarded
 const SeenUpdateExpireTime = 1 * time.Hour
 
+// MaxForwardingHops is the maximum number of times that Netceptor will forward a data packet
+const MaxForwardingHops = 30
```

Comment:
should this be configurable? We dont need that to merge this but I figure we should think about that

Reply:
Most of these constants probably ought to be configurable.